src/js/plugin/PluginEventProxy.js

```diff
@@ -0,0 +1,36 @@
+import PluginDispatcher from "./external/PluginDispatcher";
+import PluginEvents from "./external/PluginEvents";
+
+import AppsEvents from "../events/AppsEvents";
+import AppsStore from "../stores/AppsStore";
+
+import Util from "../helpers/Util";
+
+var proxies = [];
+
+function registerProxy(proxy) {
+  proxies.push(function () {
+    proxy.store.on(proxy.storeEvent, () => {
+      PluginDispatcher.dispatch({
+        eventType: proxy.pluginEvent,
+        data: Util.deepCopy(proxy.store[proxy.storeGetter])
+      });
+    });
+  });
+}
+
+registerProxy({
```

Comment:
Why not, but only nice-to-have. Don't see a real advantage.
Let's keep it simple for now.